global_segment_map_node/include/voxblox_gsm/conversions.h

```diff
@@ -1,5 +1,5 @@
-#ifndef VOXBLOX_GSM_CONVERSIONS_H_
-#define VOXBLOX_GSM_CONVERSIONS_H_
+#ifndef VOXBLOX_GSM_INCLUDE_VOXBLOX_GSM_CONVERSIONS_H_
```

Comment:
do you want to have the INCLUDE in the include guards?

Reply:
Was making it consistent with was in similar files, but you're right, I changed them all to skip the INCLUDE.